src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -956,12 +956,15 @@ public function removeCartRuleAction(int $orderId, int $orderCartRuleId): Redire
      */
     public function updateInvoiceNoteAction(int $orderId, int $orderInvoiceId, Request $request): RedirectResponse
     {
-        $this->getCommandBus()->handle(new UpdateInvoiceNoteCommand(
+        try {
+            $this->getCommandBus()->handle(new UpdateInvoiceNoteCommand(
             $orderInvoiceId,
             $request->request->get('invoice_note')
-        ));
-
-        $this->addFlash('success', $this->trans('Successful update.', 'Admin.Notifications.Success'));
+            )));
+            $this->addFlash('success', $this->trans('Successful update.', 'Admin.Notifications.Success'));
+        } catch(InvoiceException $e) {
```

Comment:
Could you check that `use PrestaShop\PrestaShop\Core\Domain\Order\Invoice\Exception\InvoiceException;` is put on the top of the file ?